plugins/base/src/main/kotlin/translators/documentables/DefaultPageCreator.kt

```diff
@@ -698,17 +824,26 @@ open class DefaultPageCreator(
     private val List<Documentable>.sourceSets: Set<DokkaSourceSet>
         get() = flatMap { it.sourceSets }.toSet()
 
+    private val List<Documentable>.dri: Set<DRI>
+        get() = map { it.dri }.toSet()
+
     private val Documentable.groupedTags: GroupedTags
         get() = documentation.flatMap { (pd, doc) ->
             doc.children.asSequence().map { pd to it }.toList()
         }.groupBy { it.second::class }
 
+    private val List<Documentable>.groupedTags: GroupedTags
+        get() = this.flatMap { it.documentation.flatMap { (pd, doc) ->
+            doc.children.asSequence().map { pd to it }.toList()
```

Comment:
Since it's only one map operation, I think `asSequence()` is not of much use here